src/modules/bulk-minter/components/Loading.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FunctionComponent } from "react";
+
+interface LoadingProps {
+    className?: string
+    w?: string
+    h?: string
+  }
+  
+const Loading: FunctionComponent<LoadingProps> = ({
+    className,
+    w = 5,
+    h = 5,
+  }) => {
+    return (
+      <div className="flex justify-center w-full">
+        <svg
+          className={`${className} animate-spin h-${w} w-${h}`}
```

Comment:
If you keep `w` and `h`, then flip them around here

```suggestion
          className={`${className} animate-spin h-${h} w-${w}`}
```